api/v1alpha1/agesecret_types.go

```diff
@@ -22,11 +22,10 @@ import (
 
 // AgeSecretSpec defines the desired state of AgeSecret
 type AgeSecretSpec struct {
-	// +kubebuilder:validation:Required
-	AgeKeyRef  string `json:"ageKeyRef"`
-	StringData string `json:"stringData"`
-	// +kubebuilder:validation:Optional
-	Suspend bool `json:"suspend"`
+	AgeKeyRef      string   `json:"ageKeyRef"`
```

Comment:
- Won't removing these markers affect existing objects?

- I think regenerating the manifests should add `suspend` to the required fields. Doesn't it?